.bmad/bmm/docs/faq.md

```
@@ -0,0 +1,587 @@
+**A:** Yes, product brief is always optional. It's most valuable for:
+
+- Level 3-4 projects needing strategic direction
+- Projects with stakeholders requiring alignment
+- Novel products needing market research
+- When you want to explore solution space before committing
+
+---
+
+## Implementation
+
+### Q: Do I need story-context for every story?
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)